package.json

```
@@ -354,6 +354,178 @@
 						}
 					}
 				}
+			},
+			{
+				"type": "reactnativedirect",
+				"label": "React Native Direct",
+				"program": "./src/debugger/direct/reactNativeDirectDebugEntryPoint.js",
+				"runtime": "node",
+				"enableBreakpointsFor": {
+					"languageIds": [
+						"javascript",
+						"typescript",
+						"javascriptreact",
+						"typescriptreact"
+					]
+				},
+				"configurationSnippets": [
+					{
+						"label": "React Native (Hermes): Debug Android",
+						"description": "%reactNative.snippets.debugAndroid.description%",
+						"body": {
+							"name": "Debug Android (Hermes)",
+							"cwd": "^\"\\${workspaceFolder}\"",
+							"type": "reactnativedirect",
+							"request": "launch",
+							"platform": "android"
+						}
+					},
+					{
+						"label": "React Native (Hermes): Attach to packager",
+						"description": "%reactNative.snippets.attachPackager.description%",
```

```suggestion
						"description": "%reactNative.snippets.attachPackagerHermes.description%",
```
Please, also update package.nls.json.